CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.0
+
+ * Removed poorly documented assets support with broken tests.
```

Comment:
Mark this as 
Breaking change:

and mention the specific API that is removed.
Also to be really nice you could write a snippet showing how to roll your own